lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -472,5 +472,37 @@ sub add_comment {
     return $self->redirect_to('test');
 }
 
+sub export {
+    my ($self) = @_;
+    $self->res->headers->content_type('text/plain');
+
+    my @groups = $self->app->schema->resultset("JobGroups")->search(undef, { order_by => 'name' });
+
+    for my $group (@groups) {
+	$self->write_chunk(sprintf("Jobs of Group '%s'\n", $group->name));
+	my @conds;
+	if ($self->param('from')) {
+	    push(@conds, { id => { '>=' => $self->param('from') } });
+	}
+	if ($self->param('to')) {
+	    push(@conds, { id => { '<' => $self->param('to') } });
+	}
+	my $jobs = $group->jobs->search({ -and => \@conds}, { order_by => 'id' } );
+	while (my $job = $jobs->next) {
+	    next if ($job->result eq OpenQA::Schema::Result::Jobs::OBSOLETED);
+	    $self->write_chunk(sprintf("Job %d: %s is %s\n", $job->id, $job->name, $job->result));
```

Comment:
"Job XXX is passed", "Job XXX is failed". Is that correct? Isn't "has" more appropriate (except for 'incomplete' jobs).